geth/params/config.go

```diff
@@ -555,42 +555,19 @@ func (c *NodeConfig) updateClusterConfig() error {
 		return nil
 	}
 
-	// TODO: Remove this thing as this is an ugly hack.
-	// Once CHT sync sub-protocol is working in LES, we will rely on it, as it provides
-	// decentralized solution. For now, in order to avoid forcing users to long sync times
-	// we use central static resource
-	type subClusterData struct {
-		Number      int      `json:"number"`
-		Hash        string   `json:"hash"`
-		StaticNodes []string `json:"staticnodes"`
-	}
-	type clusterData struct {
-		NetworkID int            `json:"networkID"`
-		Prod      subClusterData `json:"prod"`
-		Dev       subClusterData `json:"dev"`
-	}
-
-	var configFile []byte
-	var err error
-
+	var clusters []clusterData
```

Comment:
We can initialize it with `defaultClusters`.

Reply:
i think we can overwrite defaultClusters array this way